libstorage/drivers/storage/rbd/utils/utils.go

```diff
@@ -177,6 +175,34 @@ func GetMappedRBDs(ctx types.Context) (map[string]string, error) {
 	return devMap, nil
 }
 
+//ParseMappedRBDs is a two-stage parse attempt because Ceph v13 has different JSON format to previous versions
+func ParseMappedRBDs(output *[]byte) (map[string]*rbdMappedEntry, error) {
```

Comment:
This warning is coming because `ParseMappedRBDs` is capitalized. That makes it an exported func, and it doesn't need to be. Lower case the first letter and this will be fine.

I would also change it such that your input var, output, should be of type `[]byte`. There is no reason to use a pointer to a slice in Go, as a slice is already essentially a pointer.